frontend/app/routes/$lang+/_protected+/status-check+/index.tsx

```diff
@@ -47,19 +47,15 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   const t = await getFixedT(request, handle.i18nNamespaces);
   const meta = { title: t('gcweb:meta.title.template', { title: t('status-check:page-title') }) };
 
-  // Check if any needed values are undefined
-  /* TODO: If values are missing, do we want to redirect to an error page? 
-   Or perhaps include a form for the missing values?
-   Or simply stay on this page and print an erro message? */
-  if (!userInfoToken.sin || !personalInformation.clientNumber || !personalInformation.birthDate || !personalInformation.firstName || !personalInformation.lastName) {
+  if (!userInfoToken.sin || !personalInformation.clientNumber) {
     instrumentationService.countHttpStatus('status-check', 400);
     throw new Response(null, { status: 400 });
   }
 
   const applicationStatusService = getApplicationStatusService();
   const lookupService = getLookupService();
 
-  const statusId = await applicationStatusService.getStatusId(userInfoToken.sin, personalInformation.clientNumber, personalInformation.firstName, personalInformation.lastName, personalInformation.birthDate.toString());
+  const statusId = await applicationStatusService.getStatusId(userInfoToken.sin, personalInformation.clientNumber, "", "", "");
```

Comment:
We should consider changing the `getStatusId` function to optionally accept first name, last name, and date of birth. Otherwise, we would be sending empty strings to the downstream API (when they should be `null`).